src/eos-util.c

```diff
@@ -421,3 +452,142 @@ eos_updater_get_timestamp_from_branch_file_keyfile (GKeyFile *branch_file,
   *out_timestamp = g_steal_pointer (&timestamp);
   return TRUE;
 }
+
+gchar *
+eos_updater_dup_envvar_or (const gchar *envvar,
+                           const gchar *default_value)
+{
+  const gchar *value = g_getenv (envvar);
+
+  if (value != NULL)
+    return g_strdup (value);
+
+  return g_strdup (default_value);
```

Comment:
One copy is in eos-util and one in eos-autoupdater. The eos-autoupdater has its own copy, because it does not link with eos-util to avoid overlinking with ostree libraries. There is a TODO item about reorganizing the convenience libraries in the build system.